core/workspace_comment/workspace_comment_render_svg.js

```diff
@@ -30,4 +30,91 @@ goog.require('Blockly.WorkspaceCommentSvg');
 
 Blockly.WorkspaceCommentSvg.prototype.render = function() {
   console.log('Rendering a comment!');
+  var steps = [];
+
+  this.setPath_(steps, this.getHeight(), this.getWidth());
+
+  var pathString = steps.join(' ');
```

Comment:
Move lines 37-42 into `setPath`, and create the `steps` array inside `setPath`

Reply:
Done!